gcs/src/dashboard.jsx

```diff
@@ -148,9 +151,43 @@ export default function Dashboard() {
     defaultValue: defaultDataMessages,
   })
 
+  const { getSetting } = useSettings()
+
+  // Alerts
+  const { dispatchAlert, dismissAlert } = useAlerts()
+  const highestAltitudeRef = useRef(0)
+
+  function updateAltitudeAlert(msg) {
```

Comment:
rather than taking `msg` and being called separately, could use `useEffect` and recalculate whenever `telemetryData` changes 🤔

Reply:
This is alright to leave until redux is implemented